test/backgroundprocessor/BackgroundBlurProcessor.test.ts

```diff
@@ -960,4 +961,36 @@ describe('BackgroundBlurProcessor', () => {
       );
     });
   });
+
+  describe('BackgroundBlurVideoFrameProcessor', () => {
+    const spec: BackgroundFilterSpec = {};
+    const options: BackgroundBlurOptions = {
+      blurStrength: BlurStrength.MEDIUM,
```

Comment:
Change this to something besides `BlurStrength.MEDIUM`? By default, it'll use the medium so you wouldn't necessarily be able to tell if it was modified.

Reply:
Good point, I can change that. The resolveOptions would append couple more things so we would be able to tell if it was changed.